Yosemite/YosemiteTests/Mocks/Networking/Remote/MockShippingLabelRemote.swift

```diff
@@ -23,6 +23,10 @@ final class MockShippingLabelRemote {
         let shippingLabelID: Int64
     }
 
+    private struct AddressValidationResultKey: Hashable {
+        let siteID: Int64
+    }
```

Comment:
Question: that's expected limitation, since addresses don't have additional ids?
From what I understand - it won't allow to map different address validation responses within a same site id. 

I think that validation success and fail will be separated test cases any way, so not a blocker 🙂

Reply:
Yes correct. The Address Validation doesn't need any other info, this is the reason why I used just the `siteID`. 👍